kit/media/android/src/main/java/tonlabs/uikit/media/HDuplicateImageView.java

```diff
@@ -2,38 +2,26 @@
 
 import android.annotation.SuppressLint;
 import android.content.Context;
-import android.graphics.Bitmap;
+import android.graphics.drawable.BitmapDrawable;
+import android.graphics.drawable.Drawable;
+import android.view.View;
 import android.widget.ImageView;
 import androidx.annotation.Nullable;
-import com.facebook.common.executors.CallerThreadExecutor;
-import com.facebook.common.references.CloseableReference;
-import com.facebook.datasource.DataSource;
-import com.facebook.imagepipeline.core.ImagePipeline;
-import com.facebook.imagepipeline.datasource.BaseBitmapDataSubscriber;
-import com.facebook.imagepipeline.image.CloseableImage;
-import com.facebook.imagepipeline.request.ImageRequest;
-import com.facebook.imagepipeline.request.ImageRequestBuilder;
-import com.facebook.react.modules.fresco.FrescoModule;
+import com.bumptech.glide.load.resource.gif.GifDrawable;
 import com.facebook.react.uimanager.ThemedReactContext;
 import com.facebook.react.uimanager.UIManagerModule;
-import com.facebook.react.views.image.ReactImageView;
-import com.facebook.react.views.imagehelper.ImageSource;
-import java.lang.reflect.Field;
-import java.lang.reflect.InvocationTargetException;
-import java.lang.reflect.Method;
+import com.facebook.react.views.view.ReactViewGroup;
 
 @SuppressLint("AppCompatCustomView")
 public class HDuplicateImageView extends ImageView {
 
     private final ThemedReactContext context;
     private final UIManagerModule uiManagerModule;
-    private final FrescoModule frescoModule;
 
     public HDuplicateImageView(Context context) {
         super(context);
         this.context = (ThemedReactContext) context;
         uiManagerModule = this.context.getNativeModule(UIManagerModule.class);
-        frescoModule = this.context.getNativeModule(FrescoModule.class);
```

Comment:
So it looks like `fresco` wasn't really needed? (Do you remember why @savelichalex has decided to deal with it?)

Reply:
Yes, it wasn't necessary. I don't think there was a purposeful use of this package. Perhaps that solution seemed more obvious.